app/routes.py

```diff
@@ -1,2 +1,65 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
+
+planet_bp = Blueprint("planet_bp", __name__, url_prefix="/planets")
+
+@planet_bp.route("", methods=["POST", "GET"])
+def handle_planets():
+    if request.method == "POST":
+        request_body = request.get_json()
```

Comment:
Doing some validation here on the request body would be appropriate (make sure it has the required fields).